### PR TITLE
Add new functions

### PR DESCRIPTION
**Pull request by Danile71**
* RowRange
* ColRange

### CODE REVIEW

**Comment by deadprogram** [comment]
Hello, @Danile71 first of all, thanks for the pull request. It probably would have been better to add a series of PRs or commits for at least each section of code, e.g. core, dnn, etc.

However, what for sure is needed is some smoke test coverage for the newly added functions, as mentioned here: https://github.com/hybridgroup/gocv/blob/master/CONTRIBUTING.md#how-to-add-a-function-from-opencv-to-gocv

> Always add a "smoke" test for the new function being added. We are not testing OpenCV itself, but just the GoCV wrapper, so all that is needed generally is just exercising the new function. 

So basically, please just add some test coverage and this PR can land.

Thank you!

**Comment by Danile71** [comment]
Hello) I added test, all check passed)

ср, 4 дек. 2019 г. в 06:23, Ron Evans <notifications@github.com>:

**Comment by Danile71** [comment]
> Hello, @Danile71 first of all, thanks for the pull request. It probably would have been better to add a series of PRs or commits for at least each section of code, e.g. core, dnn, etc.
> 
> However, what for sure is needed is some smoke test coverage for the newly added functions, as mentioned here: https://github.com/hybridgroup/gocv/blob/master/CONTRIBUTING.md#how-to-add-a-function-from-opencv-to-gocv
> 
> > Always add a "smoke" test for the new function being added. We are not testing OpenCV itself, but just the GoCV wrapper, so all that is needed generally is just exercising the new function.
> 
> So basically, please just add some test coverage and this PR can land.
> 
> Thank you!

Hi!) All checks have passed)
